provisionersdk/proto/provisioner.proto

```diff
@@ -94,6 +94,7 @@ message Agent {
         string script = 3;
         int64 interval = 4;
         int64 timeout = 5;
+        int64 order = 6;
```

Comment:
Unfortunate that it generates an int64 here for an int32 :\

Reply:
Yeah, this is unfortunate. It is aligned with parameter order type though.